### PR TITLE
spellcheck workflow added

### PR DESCRIPTION
**Pull request by baimamboukar**

### CODE REVIEW

**Comment by baimamboukar** [comment]
Hello @bebatut !!
This is the new PR about spell check action

File: .github/workflows/spellcheck.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  build:
+    name: Spellcheck
+    runs-on: ubuntu-latest
+    steps:
+      # The checkout step
+      - uses: actions/checkout@v3
+      - uses: rojopolis/spellcheck-github-actions@0.27.0
+        name: Spellcheck
+        with:
+          source_files: "README.md"
```
Comment (bebatut) [inline .github/workflows/spellcheck.yaml:14]: It would be nice if it could check all Markdown files, not just the README

**Review by bebatut — COMMENTED** [review]
Nice. Thanks @baimamboukar

File: .github/workflows/spellcheck.yaml (hunk 1 above)
Comment (baimamboukar) [reply .github/workflows/spellcheck.yaml:14]: Hi... 
It is actually possible 😇
Should I do so?

Comment (bebatut) [reply .github/workflows/spellcheck.yaml:14]: That would be really nice

File: .spellcheck.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      # - '_includes/**/*.md'
+      # - '_posts/**/*.md'
+      # - '_ols-1/**/*.md'
+      # - '_ols-2/**/*.md'
+      # - '_ols-3/**/*.md'
+      # - '_ols-4/**/*.md'
+      # - '_ols-5/**/*.md'
+      # - '_ols-6/**/*.md'
+      # - '_ols-7/**/*.md'
+      # - '_ols-8/**/*.md'
```
Comment (bebatut) [inline .spellcheck.yml:36]:
```suggestion
       - '*.md'
      - '_includes/*.md'
       - '_posts/*.md'
       - '_ols-1/*.md'
       - '_ols-2/*.md'
       - '_ols-3/*.md'
       - '_ols-4/*.md'
       - '_ols-5/*.md'
       - '_ols-6/*.md'
       - '_ols-7/*.md'
       - '_ols-8/*.md'
```

**Pull request closed without merging**